AggregateMetrics/AggregateMetrics/AzureWebApp/DefaultCounters.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.ApplicationInsights.DataContracts;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    internal class DefaultCounters
+    {
+        private static readonly DefaultCounters instance = new DefaultCounters();
+
+        private DefaultCounters() { this.Initialize(); }
+
+        /// <summary>
+        /// Return the only instance of DefaultCounters.
+        /// </summary>
+        public static DefaultCounters Instance
```

Comment:
You can store the list in the Telemetry Module, no need to create a singleton for it